test/elixir_bench/repos/repos_test.exs

```diff
@@ -49,18 +49,23 @@ defmodule ElixirBench.ReposTest do
     test "update_repo/2 with invalid data returns error changeset" do
       repo = repo_fixture()
       assert {:error, %Ecto.Changeset{}} = Repos.update_repo(repo, @invalid_attrs)
-      assert Repos.fetch_repo_by_slug(repo.owner <> "/" <> repo.name) == {:ok, repo}
+      assert Repos.fetch_repo_by_slug(Repos.Repo.slug(repo)) == {:ok, repo}
     end
 
     test "delete_repo/1 deletes the repo" do
       repo = repo_fixture()
       assert {:ok, %Repo{}} = Repos.delete_repo(repo)
-      assert Repos.fetch_repo_by_slug(repo.owner <> "/" <> repo.name) == {:error, :not_found}
+      assert Repos.fetch_repo_by_slug(Repos.Repo.slug(repo)) == {:error, :not_found}
```

Comment:
in general one should be careful with using own application code in assertions. Because that's easily how bugs bubble up and aren't detected.

For the assertions, as I assume the name is constant we could just put the verbatim repo slug in a module attribute `@repo_slug "my-cool/repo-slug"

Reply:
Didn't get the point. You mean not to use in tests or just inside an assertion? I make use of this method in schema_test as well, should I also remove from it?

Reply:
Hey, yeah don't use it in assertion or well... mostly don't. You want your "test code" and production code not to be interdependent so much (other than the thing under test of course)

Imagine you break `Repo.slug` - all the tests using it would be broken as well!